wallet/src/de/schildbach/wallet/ui/send/SweepWalletFragment.java

```diff
@@ -458,19 +460,18 @@ private boolean utxoSpentBy(final Set<Transaction> transactions, final UTXO utxo
             public void onFail(final int messageResId, final Object... messageArgs) {
                 dismissProgress();
 
-                baseAlertDialogBuilder.setTitle(getString(R.string.sweep_wallet_fragment_request_wallet_balance_failed_title));
-                baseAlertDialogBuilder.setMessage(getString(messageResId, messageArgs));
-                baseAlertDialogBuilder.setPositiveText(getString(R.string.button_retry));
-                baseAlertDialogBuilder.setPositiveAction(
-                        () -> {
-                            requestWalletBalance();
-                            return Unit.INSTANCE;
-                        }
-                );
-                baseAlertDialogBuilder.setNegativeText(getString(R.string.button_dismiss));
-                baseAlertDialogBuilder.setShowIcon(true);
-                alertDialog = baseAlertDialogBuilder.buildAlertDialog();
-                alertDialog.show();
+                AdaptiveDialog.create(
+                        R.drawable.ic_error,
+                        getString(R.string.sweep_wallet_fragment_request_wallet_balance_failed_title),
+                        getString(messageResId, messageArgs),
+                        getString(R.string.button_close),
+                        getString(R.string.button_retry)
+                ).show(requireActivity(), (Function1<Boolean, Unit>) result -> {
+                    if (result != null && result) {
+                        requestWalletBalance();
+                    }
+                    return Unit.INSTANCE;
+                });
```

Comment:
This dialog was replaced with an adaptive dialog to fix a crash.  Other dialogs in this Fragment were not replaced.  They are not easy to test and also I figured it would be better to change these on the `master` branch first.